common/wadfile.h

```diff
@@ -83,4 +83,22 @@ typedef struct mip_s
 	unsigned int	offsets[4];	// four mip maps stored
 } mip_t;
 
+typedef struct
+{
+	char identification[4]; // "WAD3"
+	int  numlumps;
+	int  infotableofs;
+} wad3header_t;
```

Comment:
These structs are defined in `filesystem/wad.c`. You can just move them from that file to this header.